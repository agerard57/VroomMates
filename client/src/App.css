@font-face {
  font-family: "Baloo2";
  src: url("./core/fonts/baloo2/Baloo2-Regular.ttf") format("truetype");
  font-weight: 400;
  font-style: normal;
}

@font-face {
  font-family: "Baloo2";
  src: url("./core/fonts/baloo2/Baloo2-Medium.ttf") format("truetype");
  font-weight: 500;
  font-style: normal;
}

@font-face {
  font-family: "Baloo2";
  src: url("./core/fonts/baloo2/Baloo2-SemiBold.ttf") format("truetype");
  font-weight: 600;
  font-style: normal;
}

@font-face {
  font-family: "Baloo2";
  src: url("./core/fonts/baloo2/Baloo2-Bold.ttf") format("truetype");
  font-weight: 700;
  font-style: normal;
}

@font-face {
  font-family: "Baloo2";
  src: url("./core/fonts/baloo2/Baloo2-ExtraBold.ttf") format("truetype");
  font-weight: 800;
  font-style: normal;
}

.App {
  text-align: center;
      font-family: "Baloo2";
}

.App>*:first-child{
    padding-top: 10vh
}

.App-link {
  color: #61dafb;
}

h3{
    font-family: "Baloo2";
    font-weight: 600;
    text-align: left;
  }