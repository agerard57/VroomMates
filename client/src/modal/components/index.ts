export { Modal } from "./Modal";
export { ModalProvider } from "./ModalProvider";
