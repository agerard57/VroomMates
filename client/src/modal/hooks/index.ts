export { useModalManager } from "./useModalManager";
export { useModal } from "./useModal";
