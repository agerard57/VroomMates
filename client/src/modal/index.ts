export { Modal } from "./components";
export { ModalProvider } from "./components";
export { useModal } from "./hooks";

export type { SlideSpec } from "./types";
