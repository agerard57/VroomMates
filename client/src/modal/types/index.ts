export type { SlideSpec } from "./SlideSpec";
export type { UseModalManager } from "./UseModalManager";
