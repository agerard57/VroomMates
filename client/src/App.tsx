/** @jsxImportSource @emotion/react */
// eslint-disable-next-line import/no-internal-modules
import "bootstrap/dist/css/bootstrap.min.css";
import React from "react";
import { FC, Suspense } from "react";
import { ToastProvider } from "react-toast-notifications";

import "./App.css";
import { LoadingScreen } from "./loadingScreen";

const RouteManager = React.lazy(() => import("./routeManager"));

export const App: FC = () => (
  <Suspense fallback={<LoadingScreen />}>
    <ToastProvider>
      <RouteManager />
    </ToastProvider>
  </Suspense>
);
