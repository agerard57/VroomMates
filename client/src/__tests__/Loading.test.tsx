import React from "react";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";

import { App } from "../App";
import { ModalProvider } from "../modal";

it("loading screen", async () => {
  const loading = renderer
    .create(
      <div>
        <ModalProvider>
          <BrowserRouter>
            <App />
          </BrowserRouter>
        </ModalProvider>
      </div>
    )
    .toJSON();

  expect(loading).toMatchSnapshot();
});
