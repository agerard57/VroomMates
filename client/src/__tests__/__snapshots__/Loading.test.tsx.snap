// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading screen 1`] = `
<div>
  <div
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    style={
      Object {
        "bottom": 16,
        "left": 16,
        "pointerEvents": "none",
        "position": "fixed",
        "right": 16,
        "top": 16,
        "zIndex": 9999,
      }
    }
  />
  <div
    className="container css-11p4ir5-Component"
  >
    <h2
      className="css-1g3gldr-Component"
    >
      Please wait!
    </h2>
    <img
      alt="loading"
      className="css-1afbq1x-Component"
      src="loadingImage.gif"
    />
    <h2>
      We are loading :)
    </h2>
  </div>
</div>
`;
